front-end/src/components/BlinkingCursor.js

```diff
@@ -1,25 +1,23 @@
-import { useState, useEffect, useRef } from 'react';
-
-function BlinkingCursor({ currentSnippet }) {
-    //https://medium.com/@bharatgupta_2334/how-to-create-blinking-cursor-effect-using-javascript-fad099e8bcb9
-    let cursor = true;
-    let speed = 350;
+// function BlinkingCursor({ currentSnippet }) {
+//     //https://medium.com/@bharatgupta_2334/how-to-create-blinking-cursor-effect-using-javascript-fad099e8bcb9
+//     let cursor = true;
+//     let speed = 350;
     
-    setInterval(() => {
-        if(cursor) {
-            document.getElementsByClassName('cursor')[0].style.opacity = 0;
-            cursor = false;
-        } else {
-            document.getElementsByClassName('cursor')[0].style.opacity = 1;
```

Comment:
Was receiving a loop pointing to this line of code, however after commenting this line out and then the entire component out, I realized that the issue isn't related to this at all. I also commented in slack in regards to it, just being thorough on what I was referring to.